fe1-web/src/features/witness/network/messages/__tests__/WitnessRegistry.test.ts

```diff
@@ -0,0 +1,21 @@
+import { describe, it } from '@jest/globals';
+
+import { ActionType, ObjectType } from 'core/network/jsonrpc/messages';
+import { ProtocolError } from 'core/objects';
+
+import { getWitnessRegistryEntry } from '../WitnessRegistry';
+
+describe('WitnessRegistry', () => {
+  it('returns entries for valid message data input', () => {
+    getWitnessRegistryEntry({ object: ObjectType.ELECTION, action: ActionType.SETUP });
+    getWitnessRegistryEntry({ object: ObjectType.ELECTION, action: ActionType.OPEN });
+    getWitnessRegistryEntry({ object: ObjectType.ELECTION, action: ActionType.CAST_VOTE });
+    getWitnessRegistryEntry({ object: ObjectType.ELECTION, action: ActionType.END });
+  });
```

Comment:
You're not actually testing anything there, right? For it to become relevant, you should add a check to the entries, but then it means that updating the registry would imply updating the test, which is not practical.

Reply:
Well yes & no, the purpose of this test was to check whether it throws an error or not but I'll make this more explicit!